### ISSUE TITLE
openapi_3 to swagger_2 incorect nullable conversion

### ISSUE THREAD

**Issue by delfrrr** [LucyBot-Inc/api-spec-converter#213]
**test-openapi.yaml**

```
openapi: "3.0.0"
info:
  version: 1.0.0
  title: Swagger Petstore
paths:
  /pets:
    post:
      tags:
        - pets
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/Pet'
      responses:
        '201':
          description: Null response
components:
  schemas:
    Pet:
      required:
        - id
        - name
      properties:
        name:
          default: null
          type: string
          nullable: true
```

Then run 

```
npx api-spec-converter --from=openapi_3 --to=swagger_2 -s yaml ./test-openapi.yaml
```

Outputs

```
swagger: '2.0'
info:
  title: Swagger Petstore
  version: 1.0.0
paths:
  /pets:
    post:
      consumes:
        - application/json
      parameters:
        - in: body
          name: body
          schema:
            $ref: '#/definitions/Pet'
      responses:
        '201':
          description: Null response
      tags:
        - pets
definitions:
  Pet:
    properties:
      name:
        default: null
        nullable: true
        type: string
    required:
      - id
      - name
x-components: {}
```

Expected output:

```
  Pet:
    properties:
      name:
        default: null
        type: 
          - null
          - string

```

**Comment by bobby-brennan** [comment]
Thanks for the report! Happy to accept a PR here - relevant file is `lib/converters/openapi3_to_swagger2.js`

**Comment by tafm** [comment]
> **test-openapi.yaml**
> 
> ```
> openapi: "3.0.0"
> info:
>   version: 1.0.0
>   title: Swagger Petstore
> paths:
>   /pets:
>     post:
>       tags:
>         - pets
>       requestBody:
>         content:
>           application/json:
>             schema:
>               $ref: '#/components/schemas/Pet'
>       responses:
>         '201':
>           description: Null response
> components:
>   schemas:
>     Pet:
>       required:
>         - id
>         - name
>       properties:
>         name:
>           default: null
>           type: string
>           nullable: true
> ```
> 
> Then run
> 
> ```
> npx api-spec-converter --from=openapi_3 --to=swagger_2 -s yaml ./test-openapi.yaml
> ```
> 
> Outputs
> 
> ```
> swagger: '2.0'
> info:
>   title: Swagger Petstore
>   version: 1.0.0
> paths:
>   /pets:
>     post:
>       consumes:
>         - application/json
>       parameters:
>         - in: body
>           name: body
>           schema:
>             $ref: '#/definitions/Pet'
>       responses:
>         '201':
>           description: Null response
>       tags:
>         - pets
> definitions:
>   Pet:
>     properties:
>       name:
>         default: null
>         nullable: true
>         type: string
>     required:
>       - id
>       - name
> x-components: {}
> ```
> 
> Expected output:
> 
> ```
>   Pet:
>     properties:
>       name:
>         default: null
>         type: 
>           - null
>           - string
> ```

i dont think that array of types its valid swagger 2, "nullable: true" should be simply removed instead
